frontend/himachal-forest-fire-hazards/style.css

```
@@ -0,0 +1,350 @@
+:root {
+    --bg: #180f0c;
+    --bg-alt: #221512;
+    --surface: #2c1a15;
+    --line: #4a2c22;
+    --text: #f7ece5;
+    --muted: #d9beae;
+    --dim: #a98873;
+    --accent: #e2622a;
+    --accent-2: #f2a65a;
+    --heading: "Playfair Display", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)